<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemCard</class>
 <widget class="QWidget" name="ItemCard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>212</width>
    <height>100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="baseWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>212</width>
     <height>100</height>
    </rect>
   </property>
   <widget class="QLabel" name="icon">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>212</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="name">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>77</y>
      <width>202</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="using_label">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>2</y>
      <width>17</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="deleteWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>212</width>
     <height>32</height>
    </rect>
   </property>
   <widget class="QPushButton" name="deleteBtn">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>3</y>
      <width>26</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
